### PR TITLE
docs: update prerequisites for running e2e tests

### PR DESCRIPTION
**Pull request by harshilgajera-crest**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -156,6 +159,21 @@ Note: Must install docker desktop.
...
 
+Troubleshooting:
+
+1. If you face an error like this 'argparse.ArgumentError: argument -K/--keepalive: conflicting option strings: -K, --keepalive'
+
+    This is likely to happen if you have older version of PSA requriements install, to solve this try to uninstall lovely-pytest-docker and pull the latest main branch
+
+2. If while running the tests you face error as below
+
+.. code:: bash
```
Comment (kdoroszko-splunk) [inline README.rst:170]: The error message is not displayed in the preview.

**Comment by kdoroszko-splunk** [comment]
There is also another issue:
`ERROR: no match for platform in manifest: not found` (it may appear for mac users with Apple Chip - it happened to me)
The resolution here would be adding 
`platform: linux/amd64` to services in `docker-compose.yml` file. 

There are two options:
1. updating docker-compose.yml the same way it was done [https://github.com/splunk/addonfactory-repository-template/blob/main/enforce/docker-compose.yml](here) - I think it shouldn't affect non apple chips machines and will prevent apple chips machines from throwing an error. 
2. adding in the troubleshooting another point with the error.

File: README.rst (hunk 1 above)
Comment (harshilgajera-crest) [reply README.rst]: Updated

**Comment by harshilgajera-crest** [comment]
> There is also another issue: `ERROR: no match for platform in manifest: not found` (it may appear for mac users with Apple Chip - it happened to me) The resolution here would be adding `platform: linux/amd64` to services in `docker-compose.yml` file.
> 
> There are two options:
> 
> 1. updating docker-compose.yml the same way it was done [https://github.com/splunk/addonfactory-repository-template/blob/main/enforce/docker-compose.yml](here) - I think it shouldn't affect non apple chips machines and will prevent apple chips machines from throwing an error.
> 2. adding in the troubleshooting another point with the error.

Have added another step to readme for troubleshooting.

**Review by kdoroszko-splunk — APPROVED** [review]

**Pull request merged**